### PR TITLE
improve testClientGetTasks() to avoid looping all elements

### LINKED ISSUE #501: Improve test `testClientGetTasks`
This test `src/test/java/com/meilisearch/integration/TasksTest.java:54` is looping over all the available tasks in the instance, which makes my development harder! 

image

The ideal would just check one or two tests and look specifically to a value or a particular assertion, not to look into every task and do the assertions in all of them.

### PR DESCRIPTION
**Pull request by oraliahdz**
# Pull Request

## Related issue
Fixes #501 

## What does this PR do?
- improves testClientGetTasks() to avoid looping all elements

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/test/java/com/meilisearch/integration/TasksTest.java [hunk 1]
```diff
@@ -58,13 +58,13 @@ public void testClientGetTasks() throws Exception {
...
 
-            assertNotNull(task.getStatus());
             assertNotNull(task.getStatus());
             assertTrue(task.getUid() >= 0);
             assertNotNull(task.getDetails());
             if (task.getType().equals("indexDeletion")) {
                 assertNotNull(task.getDetails().getDeletedDocuments());
             }
+            break;
         }
```
Comment (alallema) [inline src/test/java/com/meilisearch/integration/TasksTest.java:68]: I think you can just something like:
```
        Task task = tasks[0];
        client.waitForTask(task.getUid());

        assertNotNull(task.getStatus());
        assertTrue(task.getUid() >= 0);
        assertNotNull(task.getDetails());
        if (task.getType().equals("indexDeletion")) {
            assertNotNull(task.getDetails().getDeletedDocuments());
        }
```
It should work

**Review by alallema — COMMENTED** [review]
Thank you for this PR @oraliahdz,
I add one suggestion for your test 😉

File: src/test/java/com/meilisearch/integration/TasksTest.java (hunk 1 above)
Comment (oraliahdz) [reply src/test/java/com/meilisearch/integration/TasksTest.java:68]: HI @alallema, thanks for the suggestion. That was my first approach, however with that I get a ArrayIndexOutOfBoundsException which is why I had to use the Break within a For Each loop

Comment (alallema) [reply src/test/java/com/meilisearch/integration/TasksTest.java:68]: This should not be the case except if this test is run before any task is processed. To make sure this does not happen, the solution is to add a task by running a `createIndex` or `AddDocument` request just before a `waitForTask()`:
```java
        String indexUid = "GetClientTasks";
        TaskInfo response = client.createIndex(indexUid);
        client.waitForTask(response.getTaskUid());
```

File: src/test/java/com/meilisearch/integration/TasksTest.java [hunk 2]
```diff
@@ -52,19 +52,21 @@ public void testClientGetTask() throws Exception {
...
-            assertTrue(task.getUid() >= 0);
-            assertNotNull(task.getDetails());
-            if (task.getType().equals("indexDeletion")) {
-                assertNotNull(task.getDetails().getDeletedDocuments());
-            }
+        assertNotNull(task.getStatus());
+        assertTrue(task.getUid() >= 0);
+        assertNotNull(task.getDetails());
+        if (task.getType().equals("indexDeletion")) {
+            assertNotNull(task.getDetails().getDeletedDocuments());
```
Comment (alallema) [inline src/test/java/com/meilisearch/integration/TasksTest.java:69]:
```suggestion
```

**Review by alallema — APPROVED** [review]
Thanks, @oraliahdz, for this PR 🎉 
LGTM! ✨

File: src/test/java/com/meilisearch/integration/TasksTest.java [hunk 3]
```diff
@@ -52,19 +52,19 @@ public void testClientGetTask() throws Exception {
...
-            assertNotNull(task.getStatus());
-            assertTrue(task.getUid() >= 0);
-            assertNotNull(task.getDetails());
-            if (task.getType().equals("indexDeletion")) {
-                assertNotNull(task.getDetails().getDeletedDocuments());
-            }
+        assertNotNull(task.getStatus());
+        assertTrue(task.getUid() >= 0);
+        assertNotNull(task.getDetails());
         }
```
Comment (alallema) [inline src/test/java/com/meilisearch/integration/TasksTest.java:68]:
```suggestion
```

**Comment by alallema** [comment]
bors merge

**Review by alallema — APPROVED** [review]

**Pull request merged**